components/toolbar/style.scss

```diff
@@ -88,8 +88,15 @@ div.components-toolbar {
 	}
 
 	// active & toggled style
-	&:not(:disabled).is-active > svg {
-		@include formatting-button-style__active;
+	&:not(:disabled).is-active {
+		& > svg,
+		&[data-subscript]:after {
```

Comment:
I don't think we need to qualify this. Couldn't `@include` simply be at the top-level for `&:not(:disabled).is-active {` ?

Reply:
These styles are quite convoluted in general 😕 Why do we apply so much effect to the SVG instead of the button.